mlflow/tracking/client.py

```diff
@@ -278,6 +740,34 @@ def log_artifact(self, run_id, local_path, artifact_path=None):
 
         :param local_path: Path to the file or directory to write.
         :param artifact_path: If provided, the directory in ``artifact_uri`` to write to.
+
+        .. code-block:: python
+            :caption: Example
+
+            from mlflow.tracking import MlflowClient
+
+            features = "rooms, zipcode, median_price, school_rating, transport"
+            with open("features.txt", 'w') as f:
+                f.write(features)
+
+            # Create a run under the default experiment id "0"
+            client = MlflowClient()
+            expermient_id = "0"
+            run = client.create_run(expermient_id)
+
+            # log and fetch the list of artifacts
```

Comment:
Does this sound like we log multiple artifacts?

Reply:
Should say fetch the artifact